NREL/SAM
# Remove code that restricts checkbox to single owner

# Pull Request Template

## Description

Remove code that limited the debt fraction checkboxes to single owner, as this code will be present in all financial models after https://github.com/NREL/ssc/pull/1208 is merged

Fixes # https://github.com/NREL/SAM/issues/1708

To test:
1. All financial models with DSCR options should have this checkbox
2. The checkbox should work where it is present. See https://github.com/NREL/ssc/pull/1208 for rough instructions. Financial models that are reasonably profitable by default might need input adjustments to test the lower bound.

## Type of change




## Checklist:

If you have added a new compute module in a SSC pull request related to this one, be sure to check the [Process Requirements](https://github.com/NREL/SAM/wiki/Compute-modules-in-SAM).
